examples/minio-bucket-secret.yaml

```diff
@@ -0,0 +1,16 @@
+# Create the AWS bucket secrets in the boot namespace
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: single-s3-creds
+  namespace: boot
+data:
+  aws_access_key_id: bWluaW9hZG1pbg==
+  aws_secret_access_key: bWluaW9hZG1pbg==
+  aws_region: dXMtd2VzdC0y
+  # endpoint: aHR0cDovL21pbmlvLXNlcnZpY2UubWluaW8tZW52LnN2Yy5jbHVzdGVyLmxvY2FsOjkwMDA=
+  endpoint: bWluaW8tc2VydmljZS5taW5pby1lbnYuc3ZjLmNsdXN0ZXIubG9jYWw=
+  # bucket: c3RhZ2UtaGNtLWZyb250ZW5kLXB1c2hjYWNoZS1idWNrZXQ=
```

Comment:
Are these comments required? I'd assume you get this from your local minio instance once it's running.

Reply:
Removed. These were remnants from all my local testing.